dll/win32/shell32/wine/shellpath.c

```diff
@@ -2144,6 +2144,73 @@ HRESULT WINAPI SHGetFolderPathAndSubDirA(
     return hr;
 }
 
+static HRESULT
+CreateShellLink(
+    LPCWSTR pszLinkPath,
+    LPCWSTR pszCmd,
+    LPCWSTR pszArg,
+    LPCWSTR pszDir,
+    LPCWSTR pszIconPath,
+    INT iIconNr,
+    LPCWSTR pszComment)
+{
+    IShellLinkW *psl;
+    IPersistFile *ppf;
+
+    HRESULT hr = CoCreateInstance(&CLSID_ShellLink, NULL, CLSCTX_INPROC_SERVER, &IID_IShellLink, (LPVOID*)&psl);
+
+    if (SUCCEEDED(hr))
+    {
+        hr = IShellLinkW_SetPath(psl, pszCmd);
```

Comment:
If this one failed, the shortcut is still created.
Is that intentional?

Reply:
Fixed.